src/util/applyMixins.ts

```diff
@@ -0,0 +1,7 @@
+export default function applyMixins(derivedCtor: any, baseCtors: any[]) {
+  baseCtors.forEach(baseCtor => {
+    Object.getOwnPropertyNames(baseCtor.prototype).forEach(name => {
+      derivedCtor.prototype[name] = baseCtor.prototype[name];
+    })
+  });
+}
```

Comment:
... and I know it's only in the tests... but that's *for now*. This is the sort of thing that could be used elsewhere. Again, micro optimizing, even premature optimizing, but perf is our biggest mandate so I'm erring in favor of that.